lib/utils.pm

```diff
@@ -1022,4 +1023,22 @@ sub svirt_host_basedir {
     return get_var('VIRSH_OPENQA_BASEDIR', '/var/lib');
 }
 
+sub prepare_ssh_localhost_key_login {
+    # in case localhost is already inside known_hosts
+    if (script_run('test -e ~/.ssh') == 0) {
```

Comment:
why check that? We rely on the directory anyway so either make sure it exists with `mkdir -p ~/.ssh` or just do not check that here and rely on it already present